genesis-tools/genesis-csv-to-json/src/csv_to_json_configs.rs

```diff
@@ -0,0 +1,64 @@
+use chrono::Utc;
+use near::config::{
+    get_initial_supply, Config, CONFIG_FILENAME, DEVELOPER_PERCENT, EXPECTED_EPOCH_LENGTH,
+    GAS_PRICE_ADJUSTMENT_RATE, GENESIS_CONFIG_FILENAME, INITIAL_GAS_LIMIT, INITIAL_GAS_PRICE,
+    MAX_INFLATION_RATE, NODE_KEY_FILE, NUM_BLOCKS_PER_YEAR, PROTOCOL_PERCENT,
+    TRANSACTION_VALIDITY_PERIOD, VALIDATOR_KEY_FILE, VALIDATOR_KICKOUT_THRESHOLD,
+};
+use near::GenesisConfig;
+use near_network::types::PROTOCOL_VERSION;
+use std::fs::File;
+use std::path::Path;
+
+const ACCOUNTS_FILE: &str = "accounts.csv";
+const NUM_SHARDS: usize = 1;
+
+/// Generates `config.json` and `genesis.config` from csv files.
+/// Verifies that `validator_key.json`, and `node_key.json` are present.
+pub fn csv_to_json_configs(home: &Path, chain_id: String) {
+    // Verify that key files exist.
+    assert!(home.join(VALIDATOR_KEY_FILE).as_path().exists(), "Validator key file should exist");
+    assert!(home.join(NODE_KEY_FILE).as_path().exists(), "Node key file should exist");
+
+    // Construct `config.json`.
+    let mut config = Config::default();
+    config.telemetry.endpoints.push(near::config::DEFAULT_TELEMETRY_URL.to_string());
```

Comment:
This line causes https://github.com/nearprotocol/stakewars/issues/41

I belive, it is safe to just remove this line here and let the `scripts/start_*.py` do the work (which is already does).

Reply:
Good catch. @frol Could you submit a fix?

Reply:
I have created a tracking issue: https://github.com/nearprotocol/nearcore/issues/1788 marked it as P2